setup.js

```diff
@@ -0,0 +1,6 @@
+const { config } = require('dotenv');
+config();
+
+const client = require('./src/utils/db.js');
+
+client.sync({ force: true })
```

Comment:
Using `sync({ force: true })` will drop and recreate all tables in the database. Ensure this is intended, as it will result in data loss. Consider using this only in development or testing environments.